packages/shared/src/components/squads/Comment.tsx

```diff
@@ -70,21 +82,22 @@ export function SquadComment({
     {
       onSuccess: (postByUrl) => {
         if (postByUrl) {
-          onUpdateForm(postByUrl);
+          onUpdateForm({ post: { post: postByUrl } });
           textinput?.current?.focus();
         }
       },
-      onError: (err: ApiErrorResult) => {
+      onError: (err: ApiErrorResult, url) => {
         if (
           err?.response?.errors?.[0].extensions.code === ApiError.NotFound &&
           !isNullOrUndefined(postItem)
         ) {
-          onUpdateForm(null);
+          onUpdateForm({ post: { post: null } });
+          getPrivateLink({ url, sourceId });
         }
       },
     },
   );
-  const [checkUrl] = useDebounce(getPost, 200);
+  const [checkUrl] = useDebounce(getPost, 1000);
```

Comment:
The request is rather expensive I believe. Do you think we should keep it at 200ms or 1000ms?

Reply:
Good point, I think test it out on preview to see how it feels and go from there.

Reply:
Good idea. Let's do that once merged.

Reply:
Current feels good to me 👍